Source/Manager/IdentityAPI/IdentityAPI.swift

```diff
@@ -229,6 +229,16 @@ class IdentityAPI {
         requestWithRetries(router: .updateProfile(userID: userID), formData: formData, headers: [.authorization: oauthToken.bearer], completion: completion)
     }
 
+    internal func updateUserDevice(
+        userID: String,
```

Comment:
Since the `/devices` endpoint no longer has the user id as a route param this can be removed.